suite-common/wallet-core/src/accounts/accountsThunks.ts

```diff
@@ -75,11 +75,13 @@ const fetchAccountTokens = async (account: Account, payloadTokens: AccountInfo['
 // as we usually want to update all accounts for a single coin at once
 export const fetchAndUpdateAccountThunk = createThunk(
     `${actionPrefix}/fetchAndUpdateAccountThunk`,
-    async (account: Account, { dispatch, extra, getState }) => {
+    async ({ accountKey }: { accountKey: AccountKey }, { dispatch, extra, getState }) => {
         const {
             selectors: { selectDevices, selectBitcoinAmountUnit },
         } = extra;
+        const account = selectAccountByKey(getState(), accountKey);
```

Comment:
I think accountKey is different because there is somehow device included in it. But generally we are interested in selected device only.

Reply:
In this case, this might not be what `selectAccountByKey` returns, as it returns first account for the key no matter what device is it for. So it might or might not be the correct one

Reply:
accountKey is unique, there can't be two accounts with same `accountKey`. This thunk doesn't and shoudn't care about that, it's resposibility of caller to pick one or both/more of these accounts. But I get you point so I verified once more which selector was used in caller and find out it was incorrect one with unspecified device :D